test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-account_financial_risk @ git+https://github.com/OCA/credit-control.git@refs/pull/272/head#subdirectory=setup/account_financial_risk
```

Comment:
```suggestion
odoo-addon-account_financial_risk @ git+https://github.com/OCA/credit-control.git@refs/pull/292/head#subdirectory=setup/account_financial_risk
```

Reply:
#272 was superseded by #292

Reply:
Thank you for the notice, changed.

Reply:
@ellbristow if you are agree with the changes, can you aprove them? Thanks!